irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -39,6 +39,15 @@ namespace {
     return (base % hash.hex() % height).str();
   }
 
+  std::string makeRejectedTxHashIndex(
```

Comment:
I suggest a bit different naming - `storeRejectedTxHash`

Otherwise `make index` sounds like `CREATE INDEX index_name ON table_name (column1, column2, ...);` will be executed under the hood :)

Reply:
I agree with you. My intention was to keep the naming consisted with other methods here. I can add the task to provide better naming for that function